### PR TITLE
Hides frameworks for nonlevy

### PR DESCRIPTION
**Pull request by chrisfoster76**

### CODE REVIEW

File: src/SFA.DAS.EAS.Web/Orchestrators/EmployerManageApprenticeshipsOrchestrator.cs [hunk 1]
```diff
@@ -206,12 +207,14 @@ public async Task<OrchestratorResponse<ApprenticeshipDetailsViewModel>> GetAppre
 
                 apprenticeship.HashedAccountId = hashedAccountId;
 
+                var includeFrameworks = commitmentData.Commitment.ApprenticeshipEmployerTypeOnApproval != ApprenticeshipEmployerType.NonLevy;
```
Comment (Pauljgraham) [inline src/SFA.DAS.EAS.Web/Orchestrators/EmployerManageApprenticeshipsOrchestrator.cs:210]: Not part of this request, but the same question as before TransferFunded cohorts should not include frameworks

File: src/SFA.DAS.EAS.Web/Orchestrators/EmployerManageApprenticeshipsOrchestrator.cs [hunk 2]
```diff
@@ -206,12 +207,14 @@ public async Task<OrchestratorResponse<ApprenticeshipDetailsViewModel>> GetAppre
...
+
                 return new OrchestratorResponse<ExtendedApprenticeshipViewModel>
                 {
                     Data = new ExtendedApprenticeshipViewModel
                     {
                         Apprenticeship = apprenticeship,
-                        ApprenticeshipProgrammes = await GetTrainingProgrammes(true)
+                        ApprenticeshipProgrammes = await GetTrainingProgrammes(includeFrameworks)
```
Comment (Pauljgraham) [inline src/SFA.DAS.EAS.Web/Orchestrators/EmployerManageApprenticeshipsOrchestrator.cs:217]: Are there any tests needing changing to cover this logic change

Comment (Pauljgraham) [inline src/SFA.DAS.EAS.Web/Orchestrators/EmployerManageApprenticeshipsOrchestrator.cs:217]: do any tests need updating for this change. again it's V1 so there may be none

File: src/SFA.DAS.EmployerApprenticeshipsService.Web.UnitTests/Orchestrators/Mappers/WhenMappingApprenticeshipViewModel.cs [hunk 3]
```diff
@@ -190,6 +191,20 @@ public async Task ThenEndpointAssessorNameIsMapped()
...
+        {
+            var apprenticeship = new Apprenticeship();
+            var commitment = new CommitmentView{  ApprenticeshipEmployerTypeOnApproval = apprenticeshipEmployerType };
+
+            var viewModel = Sut.MapToApprenticeshipViewModel(apprenticeship, commitment);
+
+            Assert.AreEqual(commitment.ApprenticeshipEmployerTypeOnApproval, viewModel.ApprenticeshipEmployerTypeOnApproval);
+        }
+
+
```
Comment (Pauljgraham) [inline src/SFA.DAS.EmployerApprenticeshipsService.Web.UnitTests/Orchestrators/Mappers/WhenMappingApprenticeshipViewModel.cs:207]: I could fly a plane through that!

File: src/SFA.DAS.EAS.Web/Orchestrators/EmployerManageApprenticeshipsOrchestrator.cs (hunk 2 above)
Comment (chrisfoster76) [reply src/SFA.DAS.EAS.Web/Orchestrators/EmployerManageApprenticeshipsOrchestrator.cs:217]: No coverage here :-(

File: src/SFA.DAS.EAS.Web/Orchestrators/EmployerManageApprenticeshipsOrchestrator.cs (hunk 1 above)
Comment (chrisfoster76) [reply src/SFA.DAS.EAS.Web/Orchestrators/EmployerManageApprenticeshipsOrchestrator.cs:210]: Transfers are dealt with differently anyway, so no

File: src/SFA.DAS.EAS.Web/Orchestrators/EmployerManageApprenticeshipsOrchestrator.cs (hunk 2 above)
Comment (chrisfoster76) [reply src/SFA.DAS.EAS.Web/Orchestrators/EmployerManageApprenticeshipsOrchestrator.cs:217]: No coverage

File: src/SFA.DAS.EmployerApprenticeshipsService.Web.UnitTests/Orchestrators/Mappers/WhenMappingApprenticeshipViewModel.cs (hunk 3 above)
Comment (chrisfoster76) [reply src/SFA.DAS.EmployerApprenticeshipsService.Web.UnitTests/Orchestrators/Mappers/WhenMappingApprenticeshipViewModel.cs]: :-(

**Pull request merged**